src/components/shared/upload/Upload.tsx

```diff
@@ -62,7 +62,7 @@ export default function Upload(props: UploadProps) {
             tokenizerValuesDisplay[index] = tokenizerCopy;
         });
         setPrepareTokenizedValues(tokenizerValuesDisplay);
-    }, [props.uploadOptions.tokenizerValues]);
+    }, [props.uploadOptions]);
```

Comment:
I think the still present bug could be related to only passing uploadOptions as dependency here, prob better to pass still the tokenizerValues or both?

Reply:
After new set up, I also can not reproduce.